### PR TITLE
fix: Massmint not working

### LINKED ISSUE #11177: Massmint not working
### What happened?

- prod works
- canary doesn't

https://github.com/user-attachments/assets/42949a7a-6833-4730-a736-c55ce6f4ac4e

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

collection `lastIndexUsed` which is used to calculate the nft `id` was missing

- [x] Closes #11177

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

https://github.com/user-attachments/assets/ba62789b-df33-4e2f-a40e-31e8034b1e56

### CODE REVIEW

File: queries/subsquid/general/collectionForMint.graphql [hunk 1]
```diff
@@ -9,5 +9,8 @@ query collectionForMint($account: String!) {
     max
     minted: nftCount
```
Comment (vikiival) [inline queries/subsquid/general/collectionForMint.graphql:10]: using minted does not work?

**Review by vikiival — APPROVED** [review]
Just clarification, otherwise ok

File: queries/subsquid/general/collectionForMint.graphql (hunk 1 above)
Comment (hassnian) [reply queries/subsquid/general/collectionForMint.graphql:10]: it doesn't for all cases, if the collection has NFTs with non sequential `sn`(like we do with drops) you could end up trying to mint a used `sn`

**Comment by vikiival** [comment]
Can you please check the failing test?

**Comment by hassnian** [comment]
> Can you please check the failing test?

#11185

**Pull request merged**